inc/authldap.class.php

```diff
@@ -1847,13 +1847,13 @@ static function getGroupCNByDn($ldap_connection, $group_dn) {
     * @param $ldap_connection
     * @param $config_ldap
     * @param $filter
+    * @param $limitexceeded
     * @param $search_in_groups         (true by default)
     * @param $groups             array
-    * @param $limitexceeded
    **/
    static function getGroupsFromLDAP($ldap_connection, $config_ldap, $filter,
-                                     $search_in_groups=true, $groups=array(),
-                                     &$limitexceeded) {
+                                     &$limitexceeded, $search_in_groups=true,
+                                     $groups=array()) {
```

Comment:
Because $search_in_groups and $groups have default value and &$limitexceeded not have it and so not conform